

export const saveUser = async ()=>{
    try{

    } catch(error){
        console.log(error)
    }
}

export const savePost = async ()=> {
    try{

    } catch (error) {
        console.log(error);
    }
}